ccmlib/scylla_node.py

```diff
@@ -1433,6 +1436,49 @@ def do_invoke(sstables):
                 ret[sst] = do_invoke([sst])
         return ret
 
+    def dump_sstables(self,
+                      keyspace: str,
+                      column_family: str) -> list[dict[str, Any]]:
+        """dump the partitions in the specified sstables using `scylla sstable dump-data`
```

Comment:
this comment is now out-of-date, we dump all sstables from the specified keyspace and table.

Reply:
sure. revised.